packages/x-data-grid/src/tests/slots.DataGrid.test.tsx

```diff
@@ -167,7 +167,7 @@ describe('<DataGrid /> - Slots', () => {
       'MUI X: useGridRootProps should only be used inside the DataGrid, DataGridPro or DataGridPremium component.',
       reactMajor < 19 &&
         'MUI X: useGridRootProps should only be used inside the DataGrid, DataGridPro or DataGridPremium component.',
-      reactMajor < 19 && 'The above error occurred in the <ForwardRef(GridOverlay2)> component',
+      reactMajor < 19 && 'The above error occurred in the <GridOverlay> component',
```

Comment:
@romgrk it seems the transformer is not properly working in this case, do you have any idea why? 

I didn't look at the implementation, but could it be that it isn't expecting `@mui/x-internals/forwardRef`?

https://app.circleci.com/pipelines/github/mui/mui-x/91899/workflows/b561ad7d-f2b4-4eab-a128-71d7e6d26287/jobs/536465

Reply:
Indeed, the plugin wasn't written to allow multiple callees with the same name. I've opened https://github.com/mui/mui-public/pull/336 and added a commit here to test it.